.github/workflows/scripts/slack_gem_notifications/notifications_methods.rb

```diff
@@ -0,0 +1,48 @@
+# encoding: utf-8
```

Comment:
What do you think about adding the `# frozen_string_literal: true` magic comment to this file?

Reply:
I think a wonderful contributor added `# frozen_string_literal: true` to all our files and it would be a shame to take that in the wrong direction! Added ✅ and thank you!